src/makefiles/cuda_64bit.mk

```diff
@@ -5,11 +5,13 @@ ifndef CUDATKDIR
 $(error CUDATKDIR not defined.)
 endif
 
+CXXFLAGS += -DHAVE_CUDA -I$(CUDATKDIR)/include -fPIC -pthread -isystem $(OPENFSTINC) -rdynamic --verbose
```

Comment:
Finally, I don't see why we were using -fPIC in the first place in our cuda compiles, as CXXFLAGS should have already had it when we built shared objects, but oh well, doesn't matter~ Don't worry about this.